hazelcast/src/main/java/com/hazelcast/jet/impl/JobClassLoaderService.java

```diff
@@ -49,6 +49,14 @@
 
 public class JobClassLoaderService {
 
+    /*
+     * On master node there is a race between closing PMS and PS.
+     * We need to close the classloader only after both have been called.
+     * The reference counting is done in removeClassloadersForJob()
```

Comment:
minor: `removeClassloadersForJob()` reference is stale after renaming.
also, it would be better to have it linked as `{@link #tryRemoveClassloadersForJob(long)}`